tests/api/integration/test_login_company_api.py

```diff
@@ -26,13 +26,15 @@ def test_post_returns_400_with_correct_data_in_query_string_but_missing_data_in_
         response = self.client.post(
             self.url, query_string=dict(email=email, password=password)
         )
-        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.status_code, 415)
+        # reponse shows expected content type
```

Comment:
What is the reason for this comment (here and in `test_login_member_api.py`)?

Reply:
I added it since I was not sure if it is common to all error codes that the response content type shows the expected content type, or if it is rather specific to 415 error code.

Reply:
I think this info is not only returned when there was an error, but also on success. We tested it on several occasions (search `self.assertEqual(response.mimetype, "application/json")`), and I guess it's not really necessary that you test it here. If you want to leave the test I'd suggest that you delete the comment (because the line of code is self-explaining) and/or move this check into a separate test function.

Reply:
you are right, I removed the comments